src/index.js

```diff
@@ -18,8 +19,41 @@ const bestSlider = (args = {}) => {
 
   let callbacks = [];
 
-  //
-  const getSlideIndex = (changeAmount) => {};
+  // Check whether new index is allowable based on total # of slides
+  const newIndexIsOutsideOfRange = (newIndex) => {
+    const { totalSlides } = state;
+
+    return newIndex < totalSlides || newIndex > totalSlides;
+  };
+
+  // Get new slide index based on change amount
+  const getSlideIndex = (changeAmount) => {
+    const { activeSlide, totalSlides } = state;
+    const { wrapAround } = config;
+
+    let tempIndex = activeSlide + changeAmount;
+    let newIndex;
+
+    if (newIndexIsOutsideOfRange(tempIndex)) {
+      if (wrapAround) {
+        if (tempIndex < 1) {
+          newIndex = totalSlides;
+        } else {
+          newIndex = 1;
+        }
+      } else {
```

Comment:
@gcamacho079 - I was thinking we should allow changeAmounts of more than just +-1, like the ability to jump 3 slides at a time, for example - so I was envisioning some logic to take that into account when crossing the boundaries of the slider. Thoughts?

Reply:
Oh absolutely makes sense. I'll revisit this